### PR TITLE
Add verification of presentations.

### PR DESCRIPTION
**Pull request by msporny**
This PR adds the ability to verify presentations as requested in issue #5.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: api.yml [hunk 1]
```diff
@@ -100,3 +100,106 @@ paths:
...
+      operationId: verifyPresentation
+      description: Performs a series of verification checks on a provided Verifiable Presentation and returns the overall status of the checks as well as the status of each individual check. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: The presentation to be verified and a set of options to use when verifying the credential.
+              type: object
+              properties:
+                verifiableCredential:
```
Comment (peacekeeper) [inline api.yml]:
```suggestion
                verifiablePresentation:
```

File: api.yml [hunk 2]
```diff
@@ -100,3 +100,106 @@ paths:
...
+      tags:
+        - internal
+      summary: Verify a given presentation.
+      operationId: verifyPresentation
+      description: Performs a series of verification checks on a provided Verifiable Presentation and returns the overall status of the checks as well as the status of each individual check. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: The presentation to be verified and a set of options to use when verifying the credential.
```
Comment (peacekeeper) [inline api.yml]:
```suggestion
              description: The presentation to be verified and a set of options to use when verifying the presentation.
```

File: api.yml [hunk 3]
```diff
@@ -100,3 +100,106 @@ paths:
...
+                  additionalProperties: false
+                  description: The Verifiable Presentation to check.
+                  example: {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "id": "urn:uuid:2011a95e-ed4a-420f-a2f8-252701e71853",
+  "type": ["VerifiablePresentation"],
+  verifiableCredential: [{
```
Comment (peacekeeper) [inline api.yml]:
```suggestion
  "verifiableCredential": [{
```

**Review by peacekeeper — APPROVED** [review]
Looks good, just found a few bugs.

**Review by dlongley — APPROVED** [review]
Looks good enough to me for immediate needs. We'll want to revise later on down the line. I only had a minor comment on the example.

**Review by mkhraisha — APPROVED** [review]

**Review by llorllale — APPROVED** [review]

**Review by troyronda — APPROVED** [review]

**Review by mavarley — APPROVED** [review]

**Comment by msporny** [comment]
Approval from 5 companies, no objections, merging.

**Pull request merged**